src/platforms/php/guides/symfony/index.mdx

```diff
@@ -17,32 +17,106 @@ Install the `sentry/sentry-symfony` package:
 composer require sentry/sentry-symfony
 ```
 
-### Configure
+<Alert level="warning" title="Note">
+**Warning:** due to a bug in all versions lower than `6.0` of the [`SensioFrameworkExtraBundle`](https://github.com/sensiolabs/SensioFrameworkExtraBundle) bundle,
```

Comment:
@ste93cry  Should we now remove the `**Warning:**`?

```suggestion
Due to a bug in all versions lower than `6.0` of the [`SensioFrameworkExtraBundle`](https://github.com/sensiolabs/SensioFrameworkExtraBundle) bundle,
```

Reply:
Yes, the box imho is more than enough to communicate that there is something important to read. I also noticed that the markdown for some reason does not work inside the box, you can see it here https://sentry-docs-git-fork-alexander-schranz-patch-1.sentry.dev/platforms/php/guides/symfony/